### ISSUE TITLE
What languages are supported

### ISSUE THREAD

**Issue by gasyoun** [computationalstylistics/stylo#47]
Is it English only?

**Comment by christofs** [comment]
No, many languages are supported, including Chinese, Japanese, Korean or Hebrew, Arabic, Cyrillic and Coptic, as long as your texts are in the UTF-8 encoding. See details here (page 10): https://github.com/computationalstylistics/stylo_howto/raw/master/stylo_howto.pdf

**Comment by Frenzie** [comment]
It depends a bit of the specific context you're thinking of. For basic tokenization just about anything goes, but for pronoun detection you might have to add your own.

https://github.com/computationalstylistics/stylo/blob/c86fdb1eaa36807a4d8ff7cf07e338c321ab22fb/man/stylo.pronouns.Rd#L13-L18
